src/app/view/_routes/editorganization/editorganization.component.html

```diff
@@ -46,16 +46,45 @@
         <button mat-button matStepperNext type="button">Next</button>
       </div>
     </mat-step>
-    <mat-step formGroupName="2" [stepControl]="formArray?.get([2])" [optional]="true">
+    <mat-step formGroupName="2" [optional]="true">
       <ng-template matStepLabel>Admin management</ng-template>
-      <mat-form-field>
-        <mat-label>Manager</mat-label>
-        <input matInput formControlName="managerEmailCtrl" placeholder="Ex. mariorossi@gmail.com" />
+      <mat-list>
+        <div *ngFor="let admin of administrators">
+          <mat-list-item class="fieldName">{{ admin.role.viewValue }}</mat-list-item>
+          <mat-list-item class="fieldData">{{ admin.email }}</mat-list-item>
+          <button
+            mat-button
+            matTooltip="Remove from administrators"
+            aria-label="Button that delete this administrator when pressed"
+            (click)="deleteAdmin(admin)"
+          >
+            <mat-icon>delete</mat-icon>
+          </button>
+          <mat-divider></mat-divider>
+        </div>
+      </mat-list>
+      <mat-form-field class="adminType">
+        <mat-label>Admin type</mat-label>
+        <mat-select formControlName="adminRole">
+          <mat-option *ngFor="let admin of adminType" [value]="admin.value">
+            {{ admin.viewValue }}
+          </mat-option>
+        </mat-select>
       </mat-form-field>
-      <mat-form-field>
-        <mat-label>Viewer</mat-label>
-        <input matInput formControlName="viewerEmailCtrl" placeholder="Ex. giorgiobianchi@gmail.com" />
+      <mat-form-field class="adminEmail">
+        <mat-label>New admin</mat-label>
+        <input formControlName="adminEmail" matInput placeholder="Ex. mariorossi@gmail.com" />
       </mat-form-field>
+      <button
+        mat-button
+        matTooltip="Add to administrators"
+        aria-label="Button that add a new administrator when pressed"
```

Comment:
qua ci va la terza singolare ("that adds")

ma io cambierei in "button to add a new administrator"

Reply:
come sopra, è un dettaglio davvero irrilevante, ma lo cambio